### PR TITLE
fix(Table): sync between body scroll and header scroll in virtualized state

### PR DESCRIPTION
**Pull request by YossiSaadi**
this is a temp workaround until the virtualized table component would get a big revamp. this workaround sets List width to be the full width and not the parent container's width

https://monday.monday.com/boards/3532714909/pulses/6513670146

### CODE REVIEW

File: packages/core/src/components/Table/Table/Table.tsx [hunk 1]
```diff
@@ -39,16 +40,6 @@ export interface ITableProps extends VibeComponentProps {
   withoutBorder?: boolean;
 }
 
-export interface ITableContext {
```
Comment (talkor) [inline packages/core/src/components/Table/Table/Table.tsx:42]: Haven't we considered it as breaking if it's taken from dist?

File: packages/core/src/components/Table/TableVirtualizedBody/TableVirtualizedBody.tsx [hunk 2]
```diff
@@ -1,38 +1,67 @@
...
 import TableBody from "../TableBody/TableBody";
 import styles from "./TableVirtualizedBody.module.scss";
-import { ScrollDirection } from "react-window";
-import { TableContext } from "../Table/Table";
+import { FixedSizeList as List, ListChildComponentProps, ScrollDirection } from "react-window";
+import { useTable } from "../context/TableContext/TableContext";
 import { RowHeights } from "../Table/TableConsts";
+import AutoSizer from "react-virtualized-auto-sizer";
+
+type Rows = Array<Record<string, unknown> & { id: string }>;
```
Comment (talkor) [inline packages/core/src/components/Table/TableVirtualizedBody/TableVirtualizedBody.tsx:10]: Both should be exported no?

File: packages/core/src/components/Table/context/TableContext/TableContext.types.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { ITableProps } from "../../Table/Table";
+import React from "react";
+
+export interface ITableContext {
```
Comment (talkor) [inline packages/core/src/components/Table/context/TableContext/TableContext.types.ts:4]: Are these supported to be exported as well? If so, they need to be registered in Table/index.ts

**Review by talkor — APPROVED** [review]
Awesome solution! 👑

File: packages/core/src/components/Table/Table/Table.tsx (hunk 1 above)
Comment (YossiSaadi) [reply packages/core/src/components/Table/Table/Table.tsx:42]: I'm not sure tbh
I do remember we've talked about it, but I don't think we need to cover those kind of cases
we do not export the `TableContext` component anyway outside from Vibe

**Pull request merged**